### PR TITLE
Removed deprecated features

### PR DESCRIPTION
**Pull request by WouterJ**

### CODE REVIEW

**Comment by dbu** [comment]
here we have the problem with the HiearchyInterface from the odm. lets change that interface for a new odm version and bump the odm version here.

**Comment by WouterJ** [comment]
Bumped the requirements and used the `HierarchyInterface`, seems ready to merge now /cc @dbu

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 Changelog
 =========
 
-* **2019-06-18**: [BC BREAK] Removed all `*.class` parameters.
+* **2016-06-19**: [BC BREAK] Removed `setParent()`/`getParent()` from the model `MenuNode`.
```
Comment (dbu) [inline CHANGELOG.md]: i think for better DX we should say something like "Removed the deprecated methods ... . Use setParentDocument / getParentDocument instead."

Comment (WouterJ) [inline CHANGELOG.md]: added an upgrade guide instead.

**Comment by dbu** [comment]
even better. thanks wouter!

**Pull request merged**